rollup.config.js

```diff
@@ -18,6 +22,8 @@ export default {
       },
     }),
     typescript({
+      typescript: require("typescript"),
```

Comment:
Is the code style inherited from upstream here?

Reply:
What do you mean? Is is about import vs require?

Reply:
I meant double quotes; this uses a different code style than we use in Sumo internally.

Reply:
I will add prettier in separated PR to focus this one only on package to be published.